packages/create-sitecore-jss/src/templates/angular/scripts/generate-config.ts

```diff
@@ -10,14 +10,14 @@ const packageConfig = require('../package.json');
  * settings as variables into the JSS app.
  * NOTE! Any configs returned here will be written into the client-side JS bundle. DO NOT PUT SECRETS HERE.
  */
-// JSS_APP_NAME env variable has been deprecated since v.21.6, SITE_NAME should be used instead
+// JSS_APP_NAME env variable has been deprecated since v.21.6, SITECORE_SITE_NAME should be used instead
 export function generateConfig(configOverrides?: { [key: string]: unknown }, outputPath?: string) {
   const defaultConfig = {
     production: false,
     sitecoreApiHost: '',
     sitecoreApiKey: 'no-api-key-set',
     jssAppName: process.env.JSS_APP_NAME,
```

Comment:
Do we want to remove this variable? And use _sitecoreSiteName_ only. (JSS_APP_NAME will be as a fallback). Since it can be confusing